sdv/sampling/hierarchical_sampler.py

```diff
@@ -60,7 +59,7 @@ def _sample_rows(self, synthesizer, num_rows=None):
         Args:
             synthesizer (copula.multivariate.base):
                 The fitted synthesizer for the table.
-            num_rows (int):
+            num_rows (int or float):
```

Comment:
I think the correct solution would be to always pass `num_rows` as an int to this method, but since that's a more complicated change, I'm just fixing the docstrings.